apps/root/src/services/contractService.ts

```diff
@@ -70,11 +70,6 @@ export default class ContractService {
     return SMOL_DOMAIN_ADDRESS[chainId];
   }
 
-  getEarnCompanionAddress(chainId: number): Address {
```

Comment:
we should leave this one right?

Reply:
I changed it based on a previous comment on this PR:

> we are already sending the chainId here, so we could fetch the companionAddress inside this function directly
I think there is no need to add it to this service, only earnService would consume it for now, and it doesn't need to depend on contractService